src/values.rs

```diff
@@ -431,6 +433,8 @@ pub enum Value {
 
     // FIXME: Add more as we identify needs
 
+    ThinkerbellRule(String, String),
```

Comment:
Strings are ambiguous. I'd prefer if it were a `struct`. I assume this is `(name, source)`?